website/docs/reference/data-test-configs.md

```diff
@@ -275,3 +275,26 @@ tests:
 ```
 
 </File>
+
+#### Specify custom configurations for generic data tests
+
+_Currently available in dbt Cloud only. Specifying custom configurations for data tests will become available in dbt Core later this year._
```

Comment:
@runleonarun could someone on the Docs team update this sentence as-needed?

It was originally authored in https://github.com/dbt-labs/docs.getdbt.com/pull/5679 months before dbt-core v1.9. But now we need to update it.

Reply:
Done! I removed it and added "beginning in dbt v1.9," to the description.

Reply:
Note that I am not versioning this section for discoverability and because it's part of a bigger unversioned feature.